docs/user-guide/user-guide.rst

```diff
@@ -1403,6 +1467,19 @@ The user can select whether single-STIG .ckls, multi-STIG .ckls, or XCCDF files
 
 --------------------------
 
+
+Export Assets CSV File
++++++++++++++++++++++++++++++++++++++++++++++++
+
+To Export Assets as CSV, select one or more Assets (no assets selected will imply all assets), and click the "Export Assets CSV" button. The CSV export will contain the Asset Name, Description, IP, FQDN, MAC, Non-Computing, STIGs, Labels, and Metadata. The CSV will also contain the STIGs assigned to each Asset in the Collection. This file 

```

Comment:
"This file" at end of paragraph.... ?

Reply:
Please move this section up so that it follows the "import" csv section, as it does in the toolbar.
Generally, I try to follow the UI components left-to-right in the docs.